storage/storagepb/kv.proto

```diff
@@ -14,11 +14,14 @@ message KeyValue {
   int64 create_revision = 2;
   // mod_revision is the last modified revision of the key.
   int64 mod_revision = 3;
+  // tombstone_revision is the tombstone revision of the key.
+  // If the key is not tombstoned, its revision is set to zero.
+  int64 tombstone_revision = 4;
```

Comment:
> I think these two ways are similar. 

I think they are quite different at least conceptually. As I mentioned, I hope we do not have the watch layer stuff down to kv layer.